jib-core/src/integration-test/java/com/google/cloud/tools/jib/builder/BuildStepsIntegrationTest.java

```diff
@@ -42,25 +49,36 @@
 /** Integration tests for {@link BuildSteps}. */
 public class BuildStepsIntegrationTest {
 
+  /** Lists the files in the {@code resourcePath} resources directory. */
+  private static ImmutableList<Path> getFilesList(String resourcePath)
+      throws URISyntaxException, IOException {
+    try (Stream<Path> fileStream =
+        Files.list(Paths.get(Resources.getResource(resourcePath).toURI()))) {
+      return fileStream.collect(ImmutableList.toImmutableList());
+    }
+  }
+
   @ClassRule public static LocalRegistry localRegistry = new LocalRegistry(5000);
 
   private static final TestBuildLogger logger = new TestBuildLogger();
 
-  @Rule public TemporaryFolder temporaryCacheDirectory = new TemporaryFolder();
-
-  @Rule public TemporaryFolder temporaryTarOutput = new TemporaryFolder();
+  @Rule public TemporaryFolder temporaryFolder = new TemporaryFolder();
```

Comment:
I guess it's harmless to have `final`. Is the repo generally lenient about `final`?

Reply:
We should definitely have `final` where-ever applicable and have things be non-final unless absolutely necessary.